### PR TITLE
add inventory file for swiss server

### PR DESCRIPTION
**Pull request by nickpell**
@TylerSheffels @dannyroberts

### CODE REVIEW

**Comment by dannyroberts** [comment]
looks good, will let @TylerSheffels sign off

File: fab/inventory/swiss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+185.12.7.167
+
+[touchforms]
+185.12.7.167
+
+[redis]
+185.12.7.167
+
+[elasticsearch]
+185.12.7.167
```
Comment (TylerSheffels) [inline fab/inventory/swiss]: I think you need `elasticsearch_node_name=hqes1` here or you will run into trouble with the ES conf.

**Comment by TylerSheffels** [comment]
One minor comment, then g2m

**Comment by czue** [comment]
we have a swiss server? who knew...

**Comment by benrudolph** [comment]
we may want to give hostnames to the ips like in the india server. some of the naming in the ansible setup works better when it's an actual hostname

**Comment by benrudolph** [comment]
though admittedly just one server so probably doesn't matter much

File: fab/inventory/swiss (hunk 1 above)
Comment (nickpell) [inline fab/inventory/swiss]: updated

**Comment by nickpell** [comment]
@dannyroberts This should be good to go now

**Comment by proteusvacuum** [comment]
out of curiosity, what are we hosting on this server?

**Pull request merged**